Content.Client/Chemistry/UI/ReagentDispenserBoundUserInterface.cs

```diff
@@ -34,6 +35,7 @@ protected override void Open()
             _window = new()
             {
                 Title = EntMan.GetComponent<MetaDataComponent>(Owner).EntityName,
+                HelpGuidebookIds = EntMan.GetComponent<GuideHelpComponent>(Owner).Guides
```

Comment:
For people looking to add this functionality to other PRs, this is literally the only code change needed to add the guidebook to a UI as long as the entity has `GuideHelpComponent` already.